lib/registries/addon/drafts/draft/draft-registration-manager.ts

```diff
@@ -135,6 +157,22 @@ export default class DraftRegistrationManager {
             });
     }
 
+    initialize() {
+        this.initializePageManagers.perform();
+        this.initializeMetadataChangeset.perform();
+    }
```

Comment:
IMHO, abstracting this away makes this harder to grok.

Reply:
also if `{ on: 'init' }` doesn't work for these tasks, we should remove it

Reply:
☝️